src/bin/rustfmt.rs

```diff
@@ -63,8 +61,8 @@ enum Operation {
 struct CliOptions {
     skip_children: bool,
     verbose: bool,
-    write_mode: Option<WriteMode>,
-    file_lines: FileLines, // Default is all lines in all files.
+    write_mode: Option<String>,
+    file_lines: Option<String>,
```

Comment:
Yeah, it is sad. I initially tried to provide setter methods, but gave up when I found out the macro system wouldn't allow me to make `set_$i()` methods without the unstable [`concat_idents!()`](https://doc.rust-lang.org/std/macro.concat_idents.html) macro.

Would any of the following be acceptable (or anything else you can think of)? I think they're all pretty ugly, but as config setting is done in so few places, maybe the ugliness doesn't matter too much.

```rust
// 1
config.verbose(); // get
config.set().verbose(true); // set
// ... but it looks like .set() might return a set of configs, which it doesn't.

// 2
config.verbose().get();
config.verbose().set(true);
// ... but it means more noise whenever we want to access a config item.

// 3
config.verbose(); // get
SetConfig::verbose(&mut config, true); // set
// ... which is pretty weird.
```